packages/dashboard/.env.example

```diff
@@ -17,6 +16,8 @@ PORT=3000
 
 # Required if using 'elem:ganache'
 ELEMENT_CONTENT_ADDRESSABLE_STORE_SERVICE_URI='/ip4/127.0.0.1/tcp/5001' 
+ELEMENT_ANCHOR_CONTRACT=''
 ETHEREUM_RCP_URL='http://localhost:8545'
 ETHEREUM_PROVIDER=$ETHEREUM_RCP_URL
-MNEMONIC="your mnemonic phrase here"
\ No newline at end of file
+MNEMONIC="your mnemonic phrase here"
```

Comment:
I don't think `MNEMONIC` is being used, we could probably take out, same with docker-compose ref: /packages/dashboard/config/index.ts

Reply:
how are you able to "sign ropsten transactions"... without providing one?

Reply:
Copy that! Will pull it out of example and docker now!

Reply:
imo, the example config should be the same config that we use to for the test method... with real values as much as possible.

Reply:
using `ETHEREUM_MNEMONIC` instead https://github.com/transmute-industries/sidetree.js/blob/main/packages/dashboard/config/index.ts#L18